net/nextdns/Makefile

```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=nextdns
-PKG_VERSION:=1.3.1
-PKG_RELEASE:=4
+PKG_VERSION:=1.4.23
+PKG_RELEASE:=5
```

Comment:
Reset this to 1. The rule is when updating PKG_VERSION, this goes back to 1.

Reply:
@rs 
can you reset PKG_RELEASE to 1 like neheb suggests.

Reply:
Sure I'll do.